hail/python/hail/matrixtable.py

```diff
@@ -665,6 +666,54 @@ def row_key(self) -> 'StructExpression':
         """
         return self._row_key
 
+    @property
+    def test(self) -> 'StructExpression':
+        print("test")
+
+        return 5
+
+    def test2(self):
+        print("test2")
+
+    _annotation_dataset_urls = None         #Class Variable
+
+    @staticmethod
+    def annotation_dataset_urls():
+        if MatrixTable._annotation_dataset_urls is None:
+            r = requests.get("http://storage.googleapis.com/hail-datasets/datasets.json")                       #Gets Datasets.JSON from Google API
+            j = r.json()                                                                                        #Reads JSON file
+            MatrixTable._annotation_dataset_urls = {(x["name"], x["reference_genome"]): x["path"] for x in j}   #Makes Dictionary table and assigns it to MatrixTable._annotation_dataset_urls
+        return MatrixTable._annotation_dataset_urls
+
+    def annotate_rows_db(self,name):
```

Comment:
@danking 
I would strongly prefer for this to be in a file in the experimental module for now. Having it here also violates our practice of putting only general-purpose functionality in matrixtable.py/table.py, and keeping genetics-specific stuff separate.